src/test/java/org/fundacionjala/sfdc/stepdefinition/LogInSteps.java

```diff
@@ -0,0 +1,40 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceLogIn;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import java.util.Properties;
+
+/**
+ * LogInSteps.java.
+ * Class for steps logIn.feature.
+ */
+public class LogInSteps {
+    private Properties prop;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "prop" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)